src/ApiTestCase.php

```diff
@@ -102,11 +109,22 @@ public function setUpClient()
      */
     public function setUpDatabase()
     {
+        $fixtureLoaders = [];
+
         if (isset($_SERVER['IS_DOCTRINE_ORM_SUPPORTED']) && $_SERVER['IS_DOCTRINE_ORM_SUPPORTED']) {
             $this->entityManager = static::$sharedKernel->getContainer()->get('doctrine.orm.entity_manager');
             $this->entityManager->getConnection()->connect();
+            $fixtureLoaders[] = new Fixtures(new Doctrine($this->getEntityManager()), [], $this->getFixtureProcessors());
+        }
+
+        if (isset($_SERVER['IS_DOCTRINE_ODM_SUPPORTED']) && $_SERVER['IS_DOCTRINE_ODM_SUPPORTED']) {
+            $this->documentManager = static::$sharedKernel->getContainer()->get('doctrine.mongodb.odm.document_manager');
+            $this->documentManager->getConnection()->connect();
+            $fixtureLoaders[] = new Fixtures(new Doctrine($this->getDocumentManager()), [], $this->getFixtureProcessors());
+        }
 
-            $this->fixtureLoader = new Fixtures(new Doctrine($this->getEntityManager()), [], $this->getFixtureProcessors());
+        if($fixtureLoaders) {
```

Comment:
We should be as explicit as possible with comparisons in PHP -> `if(!empty($fixtureLoaders))`